stdlib/public/core/UnsafePointer.swift.gyb

```diff
@@ -157,8 +157,9 @@ public struct ${Self}<Pointee>
 
   /// Accesses the `Pointee` instance referenced by `self`.
   ///
-  /// - Precondition: the pointee has been initialized with an instance of
-  ///   type `Pointee`.
+  /// - Precondition: the pointee has been initialized with an
+  ///   instance of type `Pointee` unless this is an assignment into
+  ///   `pointee` and `Pointee` is a trivial type.
```

Comment:
You ought to be able to doc-comment getters and setters separately, but I doubt our tooling can do anything useful with that.  @natecook1000 should probably figure out the phrasing here; "this" doesn't seem like the right word to me.